### ISSUE TITLE
Implement Lavalink v3.4 features

### ISSUE THREAD

**Issue by SKProCH** [angelobreuer/Lavalink4NET#76]
**Is your feature request related to a problem? Please describe.**
In a new Lavalink update several features and changes [were introduced](https://github.com/freyacodes/Lavalink/blob/dev/IMPLEMENTATION.md#significant-changes-v33---v34):
- Added filters
- The error string on the TrackExceptionEvent has been deprecated and replaced by the exception object following the same structure as the LOAD_FAILED error on /loadtracks
- Added the connected boolean to player updates.

I'd love to see support for filters

**Comment by angelobreuer** [comment]
Hello @SKProCH,

thank you for your issue! I have implemented the changes for Lavalink v3.4. Lavalink4NET now supports filters, the `connected` property (but it is not used as Lavalink4NET had its own logic for connection tracking) and the exception event in the `/loadtracks` response. Note that the latest version of Lavalink4NET will only work with a preview version of a Lavalink server (https://ci.fredboat.com/viewLog.html?buildId=8871&buildTypeId=Lavalink_Build&tab=artifacts). The releases on the GitHub page will only work partially. Additionally, I also implemented player state capturing that can be used to rewind a player state. This is currently used by Lavalink4NET internally to move players and their state (so that if a node crashes, the player is moved to another one with the same track, filters, ...).

The filters can be used like the following:

```csharp
player.Filters.Rotation = new RotationFilterOptions()
{
    Frequency = 1F,
};

await player.Filters.CommitAsync();
```

Do not forget to commit the filter changes, otherwise they won't be applied.

The NuGet packages should be ready in a few minutes, as I have the premonition that my NuGet token expired.
